data/root/etc/inst_setup

```diff
@@ -55,7 +55,7 @@ unset SSH_FAILED
 stty sane 2>/dev/null
 
 # get hostname & hostips
-hostip_from_wicked >/tmp/host_ips 2>/tmp/host_name
+hostip_from_wicked /tmp/host_ips /tmp/host_name
```

Comment:
ehm, is it possible to split / refactor this into two functions / helper scripts to have just ```hostip_from_wicked``` and e.g. ```hostname_from_wicked```?

Moreover, I'd fix the behavior when ```host_name``` sometimes contains hostname and sometimes first ip.